media/js/firefox/sync.js

```diff
@@ -126,6 +126,12 @@
         'data-download-version': 'Firefox for Android'
     });
 
+    // Firefox Sync sign in flow button
+    $('#cta-sync').on('click', function(e) {
+        e.preventDefault();
+        Mozilla.UITour.showFirefoxAccounts();
```

Comment:
I guess there is no way to really test this with Casper as the tests are run in Phantom? Then again, would it actually be possible even if it was running in Slimer?